### PR TITLE
this edit may make the connection between the license and CoC more ro…

### PR DESCRIPTION
**Pull request by mwweinberg**
…bust

Right now I'm just adding this suggestion for discussion.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ In order to receive this license, you must agree to its rules. The rules of this
 
 ## Copyright
 
-Each contributor licenses you to do everything with this software that would otherwise infringe that contributor’s copyright in it.
+Each contributor licenses you to do everything with this software that would otherwise infringe that contributor’s copyright in it provided that: a) your use of this software complies with the ml5.js Code of Conduct as it exists at the time of your use; and b) as applicable, you cooperate with any investigations your compliance with the ml5.js Code of Conduct.
```
Comment (tieguy) [inline LICENSE.md:15]: investigations `of` your compliance?

**Comment by tieguy** [comment]
Is this redundant to/somewhat conflicting with "Excuse" and "Code of Conduct? Seems like you want either this pull request, or the original text, not both?

One potential way to get at what you're wanting here is to rename "Excuse" to make its purpose more clear, which is something like "Compliance with Conditions". So something like the following, to replace the current Excuse (note I've made some small edits that I will also propose upstream to Blue Oak, and I also have some other suggestions about this language that I'll file separately):

> Compliance with Conditions
> 
> _Notices and Code of Conduct are conditions of the licenses granted in Copyright and Patent._
> 
> If anyone notifies you in writing that you have not complied with Notices, you can keep your license_s_ by taking all practical steps to comply within 30 days after the _notification_. If you do not do so, your license ends immediately.
> 
> If anyone notifies you in writing that you have not complied with the Code of Conduct _as it exists at the time of your use_, you can keep your license by taking all practical steps to comply within 30 days after the notice. If you do not do so, and the ml5.js Code of Conduct Committee (or its equivalent or successor) agrees that you are in violation of the Code of Conduct, _or you do not cooperate with any investigation by the Committee of the alleged violation_, your license ends immediately.

**Comment by mwweinberg** [comment]
(this was a suggestion from an anonymous third party and I put it here for just this type of discussion)

I feel pretty good that the original language addresses most of this.  The @tieguy language also addresses it, but I'm not totally sure if the benefit of further modifying Blue Oak is worth whatever additional confusion that may create.

I'm also somewhat conflicted about the explicit requirement to cooperate with the investigation. If the CoC Committee is investigating an allegation of a CoC, they have the ability to revoke your license if they find that there has been a violation.  If you decide not to participate in the process, isn't that kind of on you? Shouldn't the threat of losing the license be enough to compel cooperation?  I recognize that this is balanced against the general legal value that it is usually better to make implicit things explicit.

**Comment by tieguy** [comment]
Good point about the cooperation requirement, and the threat of losing the
license being the primary incentive there rather than anything we can write
in the license. (Not sure what "cooperate" means in this context anyway.)

**Comment by mwweinberg** [comment]
To the extent 'cooperate' means 'prove an explanation as to why your facially unreasonable activity is actually quite reasonable,' one would hope that staying within the license would provide enough incentive to do it.

**Pull request closed without merging**